app/views/petition_mailer/privacy_policy_update_email.html.erb

```diff
@@ -0,0 +1,223 @@
+<p>Dear <%= @name %></p>
+
+<p>
+  We're getting in touch to let you know we’re making changes to the privacy notice for the UK Government and Parliament Petitions site (<%= link_to(home_url, home_url) %>)on 1st March 2022. We are contacting you to make you aware of these changes because you have started/signed the following petitions that will be affected by these changes:
```

Comment:
You can just use `nil` for the first argument to `link_to` here, e.g:

```
<%= link_to(nil, home_url) %>
```

Reply:
Done!

In fact I had done this before and then changed it back 🤦